<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnergySourceDialog</class>
 <widget class="QDialog" name="EnergySourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>272</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Energy source parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/wizard.png</normaloff>:/images/wizard.png</iconset>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter a name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEnergyName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelFinalEnergyConsumption">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter final energy consumption (MWh/y):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxFinalEnergyConsumption">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>99999999999999991611392.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirmParams">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources>
 </resources>
 <connections/>
</ui>
